GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+KEXTFILENAME="PrjFSKext.kext"
+VFSFORDIRECTORY="/usr/local/vfsforgit"
+PRJFSKEXTDIRECTORY="/Library/Extensions"
+GVFSCOMMANDPATH="/usr/local/bin/gvfs"
+UNINSTALLERCOMMANDPATH="/usr/local/bin/uninstall_vfsforgit.sh"
+INSTALLERPACKAGEID="com.vfsforgit.pkg"
+KEXTID="io.gvfs.PrjFSKext"
+
+function UnloadKext()
+{
+    kextLoaded=`/usr/sbin/kextstat -b "$KEXTID" | wc -l`
+    if [ $kextLoaded -eq "2" ]; then
+        unloadCmd="sudo /sbin/kextunload -b $KEXTID"
+        echo "$unloadCmd..."
+        eval $unloadCmd || exit 1
+    fi
+}
+
+function UnInstallVFSForGit()
+{
+    if [ -d "${VFSFORDIRECTORY}" ]; then
+        rmCmd="sudo /bin/rm -Rf ${VFSFORDIRECTORY}"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+    
+    if [ -d "${PRJFSKEXTDIRECTORY}/$KEXTFILENAME" ]; then
+        rmCmd="sudo /bin/rm -Rf ${PRJFSKEXTDIRECTORY}/$KEXTFILENAME"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+    
+    if [ -s "${GVFSCOMMANDPATH}" ]; then
+        rmCmd="sudo /bin/rm -Rf ${GVFSCOMMANDPATH}"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+}
+
+function ForgetPackage()
+{
+    if [ -f "/usr/sbin/pkgutil" ]; then
+        forgetCmd="sudo /usr/sbin/pkgutil --forget $INSTALLERPACKAGEID"
+        echo "$forgetCmd..."
+        eval $forgetCmd
+    fi
+}
+
+function RemoveUnInstaller()
+{
+    if [ -f "${UNINSTALLERCOMMANDPATH}" ]; then
```

Comment:
Do we still need `RemoveUnInstaller`?  I thought the uninstall script was now being installed to `/usr/local/vfsforgit`.

Also, could you please avoid force pushing to PR when they are still in progress/review (or keep your old commits when you do)?  I can't tell what change(s) you made in your last push and it makes it difficult to understand what was updated.

Reply:
Actually, I see now that if I click the "force-pushed" link in this comment:

> alameenshah force-pushed the alameenshah:master  branch from 86a8760 to 87c457e an hour ago

GitHub will show the diff, but I still can't see the old commit on it own (strange!).  In general though waiting until the PR is signed off before collapsing commits will make it easier to review.

Reply:
And I am wrong again, the old commit *cannot* be found here (remove the space between https: and /):

https: //github.com/Microsoft/VFSForGit/pull/689/commits/86a8760a4970a399a1c8b232cda7ad899d4ff40e

But it can be found here:

https: //github.com/Microsoft/VFSForGit/commit/86a8760a4970a399a1c8b232cda7ad899d4ff40e

Perhaps the old commits are being kept around?

Reply:
I was hoping that would be the last commit and so pushed a squash merge, my bad sorry.

Reply:
No worries, and as mentioned above it actually is possible to see the diff 😄